src/widgets/powered-by/powered-by.js

```diff
@@ -0,0 +1,94 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
+import cx from 'classnames';
+
+import PoweredBy from '../../components/PoweredBy/PoweredBy';
+import connectPoweredBy from '../../connectors/powered-by/connectPoweredBy.js';
+import { getContainerNode } from '../../lib/utils.js';
+import { component } from '../../lib/suit';
+
+const suit = component('PoweredBy');
+
+const renderer = ({ containerNode, cssClasses }) => (
+  { widgetParams },
+  isFirstRendering
+) => {
+  if (isFirstRendering) {
+    const { url, theme } = widgetParams;
+
+    render(
```

Comment:
It won't have a huge impact here but we don't render at the init step - only at the render.

Reply:
How would you see it then?

Reply:
At the init step you don't (see below). We only call `prepareTemplateProps` at the first render we early return right after. Otherwise we might have a flash of content with the empty state.

https://github.com/algolia/instantsearch.js/blob/92cf470442be7321b9102b8ae7585f302029ff0d/src/widgets/hits/hits.js#L16-L45

Reply:
Since this widget doesn't depend on the state and isn't meant to be re-rendered, I though it would be appropriate to only render it at first rendering. Am I wrong?